### PR TITLE
Vetclinic 1

### PR DESCRIPTION
**Pull request by Yilikal250**
- [ ] Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- [ ] Created a database named vet_clinic.
- [ ] Created a table named animals:
- [ ] Populated the table with given data.
- [ ] Copied the SQL codes used in the respective directories.

SQL PHOTOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
-# vetclinic-new
\ No newline at end of file
+# curriculum-databases-projects-template
```
Comment (rotimiazeez) [inline README.md]: - Please ensure your README.MD file is similar to the provided [Template ](https://github.com/microverseinc/readme-template)  by Microverse with all the sections filled out for a more professional README file 😉

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rotimiazeez) [inline README.md]: - Please ensure the MIT License link provided is valid as the current link is broken 😢 
Screenshot_20230222-205154_Chrome

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Yilikal250 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Git flow followed 👌🏽
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by victorokoroji — APPROVED** [review]
Hi @Yilikal250, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**